lib/components/home/artwork_rail.js

```diff
@@ -31,8 +33,8 @@ class ArtworkRail extends React.Component {
   renderExpansionButton() {
     if (!this.state.expanded) {
       return (
-        <TouchableHighlight onPress={this.onPress.bind(this) } underlayColor={'white'}>
-          <View style={styles.expansionButton}/>
+        <TouchableHighlight style={styles.expansionButton} onPress={this.onPress.bind(this) } underlayColor={'white'}>
+            <View style={{height: 10, width: 10, alignSelf: 'center'}} />
```

Comment:
Leaving this here because `TouchableHighlight` requires one (and only one) child, but this view doesn't actually do anything at the moment.